hw/ip/adc_ctrl/doc/checklist.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: "DCD Checklist"
+title: "adc_ctrl Checklist"
```

Comment:
Recommend `sed` without the `i` to search and replace ALL_CAPS and lowercase separately.

Reply:
I'm not familiar with sed. Do you want to replace adc_ctrl to ADC_CTRL? Sure. I can do it.

Reply:
The goal is to keep the current capitalization. So "DCD" should become "ACD_CTRL", and "dcd" should become "acd_ctrl" Not sure how you did the search and replace, but just do it individually for the all-caps and the lowercase version.

Reply:
Okay. Noted.